GitUI/CommandsDialogs/FormPull.cs

```diff
@@ -437,7 +437,7 @@ public DialogResult PullChanges(IWin32Window owner)
                         PopStash();
                     }
 
-                    ScriptManager.RunEventScripts(this, ScriptEvent.AfterPull);
+                    executeAfterScripts();
```

Comment:
It looks like an existing bug, I don't think we should be running scripts if there was an error or the operation was aborted.

Reply:
I updated the PR with a check if the pull was aborted or an error occured. I think if the pull results in merge conflicts this is not an error and the AfterPull scripts should run, correct me if I am missing something here.